apps/cookbook/src/app/showcase/grid-showcase/grid-showcase.component.ts

```diff
@@ -7,6 +8,21 @@ import { ApiDescriptionProperty } from '~/app/shared/api-description/api-descrip
   templateUrl: './grid-showcase.component.html',
 })
 export class GridShowcaseComponent {
+  breakpoints = DesignTokenHelper.breakpoints;
+  breakpointKeyToShorthand = new Map<string, string>([
```

Comment:
Consider strong typing to prevent typos / keep in sync if breakpoints are changed later:
```suggestion
  breakpointKeyToShorthand = new Map<keyof typeof DesignTokenHelper.breakpoints, string>([
```